### PR TITLE
Updates for Assets for Sprint 05.26

### PR DESCRIPTION
**Pull request by atangOSIsoft**

### CODE REVIEW

**Review by digitaljoshua — APPROVED** [review]

File: content/api-reference/assets/asset-search-api.md [hunk 1]
```diff
@@ -130,11 +130,13 @@ Below is a response when query string is "Name:*Tracer*".
 
 Also returned is a list of `Results`. Each result contains  
 - The `MatchProperties` - a list of matched property objects. Each `MatchProperties` object contains the matched fields and their values.
-- `Score` - number that indicates the relevancy of the match.
 - `Id` - Id of the matched asset.
 - `TypeId` - asset type Id of the asset. This is null if the asset does not reference an asset type.
 - `Name` - asset name.
 - `Description` - asset description.
+- `ETag` - Version tag.
+- `CreatedDate` - Created date of the asset.
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:138]:
```suggestion
- `CreatedDate` - Asset creation date.
```

File: content/api-reference/assets/asset-search-api.md [hunk 2]
```diff
@@ -208,8 +214,10 @@ Filter strings are not case sensitive. Numeric types must be passed as strings a
 
 | Query String                                 | Description                                                  |
 | -------------------------------------------- | ------------------------------------------------------------ |
-| filter[location]=Earth                       | Returns all assets that contain a metadata name = **location** and value is **Earth**. |
-| filter[location]=Earth&filter[device]=tracer | Returns all assets that contain a metadata name = **location** and value is **Earth** and also contains a metadata with name = **device** and value is **tracer**. |
+| filter[location]=Earth                       | Filter to ensure that only assets that contain a metadata name = **location** and value is **Earth** is returned. |
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:217]:
```suggestion
| filter[location]=Earth                       | Filter that only returns assets that contain the metadata name = **location** and the value of this metadata is **Earth**. |
```

File: content/api-reference/assets/asset-search-api.md (hunk 1 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:138]: Thanks! Looks good.

File: content/api-reference/assets/asset-search-api.md (hunk 2 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:217]: Thanks! Looks good.

File: content/api-reference/assets/asset-search-api.md [hunk 3]
```diff
@@ -208,8 +214,10 @@ Filter strings are not case sensitive. Numeric types must be passed as strings a
...
+| filter[location]=Earth&filter[device]=tracer | Filter to ensure that only assets that contain a metadata name = **location** and value is **Earth** and also contains a metadata with name = **device** and value is **tracer** is returned. |
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:218]:
```suggestion
| filter[location]=Earth&filter[device]=tracer | Filter that only returns assets that contain both of the following metadata. The first metadata name = **location** with **Earth** as the metadata value, and the second metadata is name = **device** with **tracer** as the metadata value. |
```

File: content/api-reference/assets/asset-search-api.md [hunk 4]
```diff
@@ -208,8 +214,10 @@ Filter strings are not case sensitive. Numeric types must be passed as strings a
...
+| filter[AssetTypeName]=HeaterType             | Filter to ensure that only assets with AssetTypeName HeaterType is returned. |
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:219]:
```suggestion
| filter[AssetTypeName]=HeaterType             | Filter that only returns assets with an AssetTypeName of "HeaterType." |
```

File: content/api-reference/assets/asset-search-api.md [hunk 5]
```diff
@@ -208,8 +214,10 @@ Filter strings are not case sensitive. Numeric types must be passed as strings a
...
+| filter[status]=Bad                           | Filter to ensure that only assets with a bad status. Status filters can contain "Good", "Bad", Warning" and "Unknown". |
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:220]:
```suggestion
| filter[status]=Bad                           | Filter that returns only assets with a bad status. Status filters can have the values "Good", "Bad", Warning", and "Unknown". |
```
What does the second sentence mean? Are you just telling them what the valid values might be?

File: content/api-reference/assets/asset-search-api.md [hunk 6]
```diff
@@ -298,3 +306,125 @@ Content-Type: application/json
     }
 ]
 ```
+
+## `Asset Autocomplete` 
+
+Asset autocomplete provides a way for you to query assets and retrieve a list of suggested assets based on your search criteria. The following asset properties are capable of autocomplete:
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:312]:
```suggestion
Asset autocomplete allows you to query assets and retrieve a list of suggested assets based on your search criteria. The following asset properties are capable of autocomplete:
```

File: content/api-reference/assets/asset-search-api.md [hunk 7]
```diff
@@ -298,3 +306,125 @@ Content-Type: application/json
...
+The tenant identifier
+
+`string namespaceId`  
+The namespace identifier
+
+`string term`  
+The search term that you want to search for.
+
+[optional] `int facetCount`  
+The maximum number of facet autocompleted items to return. By default, this value is 0, this means you are not interested in suggestions for facets.
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:337]:
```suggestion
The maximum number of facet autocompleted items to return. The default is 0 which means you will not get any suggestions for facets.
```

File: content/api-reference/assets/asset-search-api.md [hunk 8]
```diff
@@ -298,3 +306,125 @@ Content-Type: application/json
...
+
+[optional] `int termCount`  
+The maximum number of token autocompleted items to return. By default, this value is 0, this means you are not interested in suggestions for tokens.
```
Comment (lasato-OSI) [inline content/api-reference/assets/asset-search-api.md:340]:
```suggestion
The maximum number of token autocompleted items to return. The default is 0 which means you will not get any suggestions for tokens.
```

File: content/api-reference/assets/asset-search-api.md (hunk 3 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:218]: Thanks.

File: content/api-reference/assets/asset-search-api.md (hunk 4 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:219]: Thanks

File: content/api-reference/assets/asset-search-api.md (hunk 5 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:220]: Yup, the second sentence is mainly for that. Perhaps we should have another section for "Status Filtering"?

File: content/api-reference/assets/asset-search-api.md (hunk 6 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:312]: Looks good.

File: content/api-reference/assets/asset-search-api.md (hunk 7 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:337]: Thanks.,

File: content/api-reference/assets/asset-search-api.md (hunk 8 above)
Comment (atangOSIsoft) [reply content/api-reference/assets/asset-search-api.md:340]: Thanks

**Pull request merged**